### DISCUSSION TITLE
有个问题想咨询下,就是docker部署的higress, 怎么修改配置参数呢

### DISCUSSION THREAD

**Discussion by MrOldPeng** [higress-group/higress#1215] [Q&A]
比如全局参数 global.defaultUpstreamConcurrencyThreshold
要如何修改

**Comment by johnlanni** [comment]
https://higress.io/docs/latest/user/configurations/
这里的配置仅局限于 k8s 下部署，因为本身是基于 helm values的配置项，具体可以看下helm下这个配置变成了什么环境变量，加上对应的变量即可。
